src/DynamoCore/Core/ICustomNodeManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Dynamo.Models;
```

Comment:
Let's move this to the `Interfaces` directory and update the namespace to reflect the move.